code/game/turfs/open.dm

```diff
@@ -119,8 +142,10 @@
 					step(C, olddir)
 					C.spin(1,1)
 		else if(lube&SLIDE_ICE)
+			C.slipping = TRUE
 			spawn(1)
 				step(C, olddir)
```

Comment:
i think it's because current slipping code just slips you until you can't slip no more